.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```

Comment:
please use a proper gitignore file to eliminate these files from version control, if necessary